pkg/operator/ceph/cluster/watcher.go

```diff
@@ -78,14 +90,15 @@ func (c *clientCluster) onK8sNode(object runtime.Object) bool {
 		return false
 	}
 
+	logger.Debugf("adding node %q to nodesCheckedForReconcile", node.Name)
```

Comment:
How about this message instead?
```suggestion
	logger.Debugf("node %q is ready, checking if it is valid for OSDs", node.Name)
```